aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -41,6 +41,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.ristretto255.basepoint_double_mul, "ristretto255.basepoint_double_mul", 440_000 * MUL],
 
     [.ristretto255.point_add, "ristretto255.point_add", 2_135 * MUL],
+    [.ristretto255.point_clone, optional "ristretto255.point_clone", 150 * MUL],
```

Comment:
These gas costs are out in the wrong format and need to be fixed. There's no `optional` keyword anymore.